src/fourcipp/fourc_input.py

```diff
@@ -284,6 +284,43 @@ def overwrite_sections(self, other: dict | FourCInput) -> None:
         else:
             raise TypeError(f"Cannot overwrite sections from {type(other)}.")
 
+    def apply_user_defaults(self, default_path: Path) -> None:
+        """Combines two Inputs by overwriting current values by a file
+        containing user defaults.
+
+        This function checks whether values exist in both objects and overwrites the current by the other.
+        At this time only top level section parameters of simple types (int, float, str, bool, None) are supported.
+
+        Args:
+            default_path: Path to the YAML file containing user defaults
+        """
+        default_input = FourCInput.from_4C_yaml(default_path, header_only=True)
+        default_sections = default_input.sections
+
+        for sectionkey in default_sections:
```

Comment:
We use camel case as recommend by pep8
```suggestion
        for section_key in default_sections:
```